### DISCUSSION TITLE
I've been checking for a whole day, and I thought it was an issue with the litellm package. I didn't expect to upgrade it to 0.30.1, and the process would restart indefinitely

### DISCUSSION THREAD

**Discussion by zouhuigang** [Kludex/uvicorn#2378] [Q&A]
INFO:     Started server process [2832190]
INFO:     Waiting for application startup.
INFO:     Application startup complete.
INFO:     Waiting for child process [2832717]
INFO:     Child process [2832717] died
INFO:     Waiting for child process [2832717]
INFO:     Child process [2832717] died

**Comment by Kludex** [comment]
Please provide an MRE.
